utils/resolvconf.go

```diff
@@ -47,6 +45,8 @@ func getResolvSearchDomains(resolvConf []byte) []string {
 	var (
 		domains []string
 		lines   [][]byte
+
+		searchRegexp = regexp.MustCompile(`^\s*search\s*(([^\s]+\s*)*)$`)
```

Comment:
Even though it should be garbage collected once it is out of scope, why is this being created for every getResolvSearchDomains() call instead of creating it once?

Reply:
I checked how often the function was called and it was only called once during the initial configuration, so my initial intention was to save stack space allocation